test/custom_operator/op.cpp

```diff
@@ -0,0 +1,37 @@
+#include <torch/op.h>
+
+#if defined(WITH_PYTHON_OP_BINDINGS)
+#include <pybind11/pybind11.h>
+#endif
+
+#include <cstddef>
+#include <vector>
+
+std::vector<at::Tensor> custom_op(
+    at::Tensor tensor,
+    double scalar,
+    int64_t repeat) {
+  std::vector<at::Tensor> output;
+  output.reserve(repeat);
+  for (int64_t i = 0; i < repeat; ++i) {
+    output.push_back(tensor * scalar);
+  }
+  return output;
+}
+
+torch::RegisterOperators registry("custom::op", &custom_op);
```

Comment:
put it into anonymous namespace? also it might be good to have a macro like REGISTER_CPU_OPERATOR in Caffe2 that takes care of the temporary variable name for you (again, we should just move that part to ATen/core and use verbatim probably)

Reply:
I think @zdevito and I would prefer fewer macros. The code looks pretty simple and readable to me :)